packages/editor-ui/src/components/ExternalSecretsProviderImage.ee.vue

```diff
@@ -1,10 +1,16 @@
 <script lang="ts" setup>
+/* eslint-disable @typescript-eslint/ban-ts-comment */
 import type { PropType } from 'vue';
 import type { ExternalSecretsProvider } from '@/Interface';
 import { computed } from 'vue';
+
+// @ts-expect-error
```

Comment:
`expect-error` does not feel right here. Could we follow something similar to here `declare module '*.svg'`? https://stackoverflow.com/questions/52759220/importing-images-in-typescript-react-cannot-find-module

Though in my quick testing, it did not work

Reply:
I tried that too. For some reason `vue-tsc` doesn't pick up the shims file. 🥲

Reply:
Maybe a TODO comment here?